doc/designs/quic-design/connection-id-cache.md

```diff
@@ -221,17 +225,41 @@ Additional status and source information is also included.
 typedef struct quic_route_st QUIC_ROUTE;
 typedef struct quic_route_table QUIC_ROUTE_TABLE;
 
-typedef struct quic_route_st {
-    QUIC_CONN_ID local;
-    QUIC_CONN_ID remote;
-    uint64_t     seq_no;        /* Sequence number for both ends */
-    unsigned int retired : 1;   /* Connection ID has been retired */
+struct quic_route_st {
+    QUIC_CONNECTION *conn;
+    QUIC_CONN_ID     local;
+    QUIC_CONN_ID     remote;
+    uint64_t         seq_no;        /* Sequence number for both ends */
+    unsigned int     retired : 1;   /* Connection ID has been retired */
 #if 0
     /* Later will require */
-    BIO_ADDR remote_address;    /* remote source address */
-    /* which exposes union bio_addr_st more widely */
+    BIO_ADDR         remote_address;/* remote source address */
 #endif
 };
+
+QUIC_ROUTE_TABLE *ossl_quic_route_new(void);
```

Comment:
`ossl_quic_route_table_new`?

Reply:
I was tossing up between route and route_table....
I choose poorly.